src/Models/Conversation.php

```diff
@@ -53,6 +56,21 @@ public function users(): BelongsToMany
         return $this->belongsToMany(config('gossip.user_model'));
     }
 
+    /**
+     * Scope a query to only include conversations of the given user.
+     *
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  \App\Models\User  $user
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function scopeOfUser($query, $user)
```

Comment:
You could typehint this with the Eloquent `Model` and than call `$user->getKey()` to retrieve ID.